### DISCUSSION TITLE
Formily Field模型与Json 互转问题

### DISCUSSION THREAD

**Discussion by beita-wang** [alibaba/formily#3897] [Q&A 2.x]
![image](https://github.com/alibaba/formily/assets/58249740/b6533c66-2746-4716-bb65-4194f0883016)

- 业务场景：我们的业务场景是有一个Select，点击Select里的选项后，去请求后端，后端会返回该页面的Json Schema，**在页面进行替换渲染。并且我们需要将用户点击过的选项生成的Json Schema页面数据缓存下来。**
- 存在的问题：
   1. 我使用的是RecursionField自己定义了一个Container容器，所以后端返回Json Schema，我直接用Field.setComponentProps直接替换页面Json Schema片段是可以的。但是有些页面我拿不到Json Schema，只能拿到Field。请问Field怎么与Json 互转。因为Field如果能转成Json Schema，我就可以直接设置了。
   ![image](https://github.com/alibaba/formily/assets/58249740/a92c6596-35ee-429a-9f96-a20daf096d37)

**Comment by liweiting1995** [comment]
这个问题你们最后解决了吗？
我也遇到了同样的问题
